src/npm.rs

```diff
@@ -0,0 +1,32 @@
+use emoji;
+use std::process::Command;
+
+pub fn npm_pack(path: &str) {
+    let pkg_file_path = format!("{}/pkg", path);
+    let output = Command::new("npm")
+        .current_dir(pkg_file_path)
+        .arg("pack")
+        .output()
+        .unwrap_or_else(|e| panic!("{} failed to execute process: {}", emoji::ERROR, e));
```

Comment:
I don't think we should be panicking here since main allows propogating up errors. I would refactor this method to do `map_err` for this error message, then use ? to propogate it up and make the method return a Result<(), failure::Error>

Reply:
I just copied the behavior from existing code, this is copy and pasted

Reply:
Oh is it? Hmm then this might need to be a separate Pr then to just go through and change it all. :) I'll leave that call up to @ashleygwilliams

Reply:
yeah let's merge this as is and then do a refactor that hits all the points. PRs welcome :D